src/server/pages.js

```diff
@@ -42,7 +42,7 @@ pages
   .add('createUpdate', '/:collectiveSlug/updates/new')
   .add('updates', '/:collectiveSlug/updates')
   .add('update', '/:collectiveSlug/updates/:updateSlug')
-  .add('createExpense', '/:parentCollectiveSlug?/:type(events)?/:collectiveSlug/expenses/new')
```

Comment:
Why renaming that?

Reply:
> Check the CI results below, it will help you to fix some parts. You can remove `src/components/__tests__/SignInUp.test.js` safely.
> 
> The magic command to update the translations files for the `update-langs` test is `npm run build:langs`

 Yes I thought of removing 'SignInUp.test.js' but as it was not mentioned anywhere in the issue so I held back. Will do that now.

Reply:
> Why renaming that
 
Will change it back. I was just checking out something and forgot to change it before commiting.